tv_apps/smart-system/js/homescreen_launcher.js

```diff
@@ -24,6 +25,16 @@
   var HomescreenLauncher = function() {
   };
 
+  HomescreenLauncher.EVENTS = [
```

Comment:
Due to the utilization of BaseModule's `EVENTS` object here, the original `handleEvent` method [1] will be overwritten by BaseModule [2]. It turns out that the events don't seem to be handled by `HomescreenLauncher` after this patch. I guess we need to replace its name with `_pre_handleEvent / _post_handleEvent` or follow the convention of `_handle_<event name>` to address each single event.

[1] https://github.com/albertopq/gaia/blob/1244682-base-module-launchers/tv_apps/smart-system/js/homescreen_launcher.js#L133
[2] https://github.com/mozilla-b2g/gaia/blob/master/tv_apps/smart-system/js/base_module.js#L333